scripts/pi-hole/php/savesettings.php

```diff
@@ -86,19 +86,36 @@ function readStaticLeasesFile()
 	return true;
 }
 
+function isequal(&$argument, &$compareto) {
+	$ret = false;
+	if(isset($argument))
+	{
+		if($argument === $compareto)
+		{
+			return true;
+		}
+	}
+	return false;
+}
+
+function isinserverlist($ip) {
+	global $DNSserverslist;
+	foreach ($DNSserverslist as $key => $value) {
+		if (isequal($value['v4p'],$ip) || isequal($value['v4s'],$ip))
+			return true;
+		if (isequal($value['v6p'],$ip) || isequal($value['v6s'],$ip))
+			return true;
+	}
+	return false;
+}
+
 	$DNSserverslist = [
-			"8.8.8.8" => "Google (Primary)",
-			"208.67.222.222" => "OpenDNS (Primary)",
-			"4.2.2.1" => "Level3 (Primary)",
-			"199.85.126.10" => "Norton (Primary)",
-			"8.26.56.26" => "Comodo (Primary)",
-			"84.200.69.80" => "DNS.WATCH (Primary)",
-			"8.8.4.4" => "Google (Secondary)",
-			"208.67.220.220" => "OpenDNS (Secondary)",
-			"4.2.2.2" => "Level3 (Secondary)",
-			"199.85.127.10" => "Norton (Secondary)",
-			"8.20.247.20" => "Comodo (Secondary)",
-        	"84.200.70.40" => "DNS.WATCH (Secondary)",
+			"Google" => ["v4p" => "8.8.8.8","v4s" => "8.8.4.4", "v6p" => "2001:4860:4860:0:0:0:0:8888", "v6s" => "2001:4860:4860:0:0:0:0:8844"],
+			"OpenDNS" => ["v4p" => "208.67.222.222", "v4s" => "208.67.220.220"],
```

Comment:
[OpenDNS also has IPv6 servers.](https://www.opendns.com/about/innovations/ipv6/) Is there any reason why they are not implemented?